src/core/include/openvino/op/util/read_value_base.hpp

```diff
@@ -16,6 +16,7 @@ class OPENVINO_API ReadValueBase : public Op, public VariableExtension {
     BWDCMP_RTTI_DECLARATION;
 
     ReadValueBase() = default;
+    virtual ~ReadValueBase() {};
```

Comment:
Is this change needed? This class is derived from VariableExtension, if base class has virtual destructor - no need to specify this in derived classes. If it is needed, probably current code style will require to use `~ReadValueBase() override {}` style

Reply:
Deleted